components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1308,6 +1339,7 @@ private void addSubResolutions() throws IOException, FormatException {
         }
         core.add(s, c);
       }
+      rs.close();
```

Comment:
This line currently conflicts with changes added in https://github.com/openmicroscopy/bioformats/pull/3276/files#diff-b6bbe95ea3520ea325f6313eb1cc47a7R1270

Reply:
I'm happy for this PR to be temporarily excluded in favor of #3276, if that makes things easier.

Reply:
@melissalinkert I am going to exclude temporarily this PR to test the build